<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SnakeViz</class>
 <widget class="QWidget" name="SnakeViz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>197</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QRadioButton" name="radioButton">
     <property name="text">
      <string>XY Plane</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QRadioButton" name="radioButton_2">
     <property name="text">
      <string>XZ Plane</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QRadioButton" name="radioButton_3">
     <property name="text">
      <string>YZ Plane</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="0" column="5">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>197</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_1">
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string> pi</string>
     </property>
     <property name="minimum">
      <double>-2.000000000000000</double>
     </property>
     <property name="maximum">
      <double>2.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.083333000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="16" colspan="5">
    <widget class="Projections" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>680</width>
       <height>680</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string> pi</string>
     </property>
     <property name="minimum">
      <double>-2.000000000000000</double>
     </property>
     <property name="maximum">
      <double>2.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.083333000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string> pi</string>
     </property>
     <property name="minimum">
      <double>-2.000000000000000</double>
     </property>
     <property name="maximum">
      <double>2.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.083333000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string> pi</string>
     </property>
     <property name="minimum">
      <double>-2.000000000000000</double>
     </property>
     <property name="maximum">
      <double>2.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.083333000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_5">
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string> pi</string>
     </property>
     <property name="minimum">
      <double>-2.000000000000000</double>
     </property>
     <property name="maximum">
      <double>2.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.083333000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_6">
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string> pi</string>
     </property>
     <property name="minimum">
      <double>-2.000000000000000</double>
     </property>
     <property name="maximum">
      <double>2.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.083333000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_7">
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string> pi</string>
     </property>
     <property name="minimum">
      <double>-2.000000000000000</double>
     </property>
     <property name="maximum">
      <double>2.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.083333000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_8">
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string> pi</string>
     </property>
     <property name="minimum">
      <double>-2.000000000000000</double>
     </property>
     <property name="maximum">
      <double>2.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.083333000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_9">
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string> pi</string>
     </property>
     <property name="minimum">
      <double>-2.000000000000000</double>
     </property>
     <property name="maximum">
      <double>2.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.083333000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_10">
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string> pi</string>
     </property>
     <property name="minimum">
      <double>-2.000000000000000</double>
     </property>
     <property name="maximum">
      <double>2.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.083333000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_11">
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string> pi</string>
     </property>
     <property name="minimum">
      <double>-2.000000000000000</double>
     </property>
     <property name="maximum">
      <double>2.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.083333000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_12">
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string> pi</string>
     </property>
     <property name="minimum">
      <double>-2.000000000000000</double>
     </property>
     <property name="maximum">
      <double>2.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.083333000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_13">
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string> pi</string>
     </property>
     <property name="minimum">
      <double>-2.000000000000000</double>
     </property>
     <property name="maximum">
      <double>2.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.083333000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_14">
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string> pi</string>
     </property>
     <property name="minimum">
      <double>-2.000000000000000</double>
     </property>
     <property name="maximum">
      <double>2.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.083333000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_15">
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string> pi</string>
     </property>
     <property name="minimum">
      <double>-2.000000000000000</double>
     </property>
     <property name="maximum">
      <double>2.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.083333000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="16" column="0">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_16">
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string> pi</string>
     </property>
     <property name="minimum">
      <double>-2.000000000000000</double>
     </property>
     <property name="maximum">
      <double>2.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.083333000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Projections</class>
   <extends>QWidget</extends>
   <header>projections.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButton</tabstop>
  <tabstop>doubleSpinBox_1</tabstop>
  <tabstop>doubleSpinBox_2</tabstop>
  <tabstop>doubleSpinBox_3</tabstop>
  <tabstop>doubleSpinBox_4</tabstop>
  <tabstop>doubleSpinBox_5</tabstop>
  <tabstop>doubleSpinBox_6</tabstop>
  <tabstop>doubleSpinBox_7</tabstop>
  <tabstop>doubleSpinBox_8</tabstop>
  <tabstop>doubleSpinBox_9</tabstop>
  <tabstop>doubleSpinBox_10</tabstop>
  <tabstop>doubleSpinBox_11</tabstop>
  <tabstop>doubleSpinBox_12</tabstop>
  <tabstop>doubleSpinBox_13</tabstop>
  <tabstop>doubleSpinBox_14</tabstop>
  <tabstop>doubleSpinBox_15</tabstop>
  <tabstop>doubleSpinBox_16</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>SnakeViz</receiver>
   <slot>refresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_1</sender>
   <signal>valueChanged(double)</signal>
   <receiver>SnakeViz</receiver>
   <slot>refresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_2</sender>
   <signal>valueChanged(double)</signal>
   <receiver>SnakeViz</receiver>
   <slot>refresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_3</sender>
   <signal>valueChanged(double)</signal>
   <receiver>SnakeViz</receiver>
   <slot>refresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_4</sender>
   <signal>valueChanged(double)</signal>
   <receiver>SnakeViz</receiver>
   <slot>refresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_5</sender>
   <signal>valueChanged(double)</signal>
   <receiver>SnakeViz</receiver>
   <slot>refresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_6</sender>
   <signal>valueChanged(double)</signal>
   <receiver>SnakeViz</receiver>
   <slot>refresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_7</sender>
   <signal>valueChanged(double)</signal>
   <receiver>SnakeViz</receiver>
   <slot>refresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_8</sender>
   <signal>valueChanged(double)</signal>
   <receiver>SnakeViz</receiver>
   <slot>refresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>408</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_9</sender>
   <signal>valueChanged(double)</signal>
   <receiver>SnakeViz</receiver>
   <slot>refresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>448</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_10</sender>
   <signal>valueChanged(double)</signal>
   <receiver>SnakeViz</receiver>
   <slot>refresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>492</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_11</sender>
   <signal>valueChanged(double)</signal>
   <receiver>SnakeViz</receiver>
   <slot>refresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>524</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>544</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_12</sender>
   <signal>valueChanged(double)</signal>
   <receiver>SnakeViz</receiver>
   <slot>refresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>574</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>578</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_13</sender>
   <signal>valueChanged(double)</signal>
   <receiver>SnakeViz</receiver>
   <slot>refresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>614</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>620</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_14</sender>
   <signal>valueChanged(double)</signal>
   <receiver>SnakeViz</receiver>
   <slot>refresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>659</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>666</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_15</sender>
   <signal>valueChanged(double)</signal>
   <receiver>SnakeViz</receiver>
   <slot>refresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>702</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>702</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_16</sender>
   <signal>valueChanged(double)</signal>
   <receiver>SnakeViz</receiver>
   <slot>refresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>754</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>768</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>clicked()</signal>
   <receiver>SnakeViz</receiver>
   <slot>refresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>clicked()</signal>
   <receiver>SnakeViz</receiver>
   <slot>refresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_3</sender>
   <signal>clicked()</signal>
   <receiver>SnakeViz</receiver>
   <slot>refresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>543</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>548</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refresh()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
